lib/services/keyVault/lib/keyVaultClient.d.ts

```diff
@@ -1223,6 +1221,76 @@ declare class KeyVaultClient {
          * 
          * @param {object} [options] Optional Parameters.
          * 
+         * @param {object} [options.certificatePolicy] The management policy for the
+         * certificate
+         * 
+         * @param {object} [options.certificatePolicy.keyProperties] Properties of the
+         * key backing a certificate.
+         * 
+         * @param {boolean} [options.certificatePolicy.keyProperties.exportable]
+         * Indicates if the private key can be exported.
+         * 
+         * @param {string} [options.certificatePolicy.keyProperties.keyType] The key
+         * type.
+         * 
+         * @param {number} [options.certificatePolicy.keyProperties.keySize] The key
+         * size in bytes. e.g. 1024 or 2048.
+         * 
+         * @param {boolean} [options.certificatePolicy.keyProperties.reuseKey]
+         * Indicates if the same key pair will be used on certificate renewal.
+         * 
+         * @param {object} [options.certificatePolicy.secretProperties] Properties of
+         * the secret backing a certificate.
+         * 
+         * @param {string} [options.certificatePolicy.secretProperties.contentType]
+         * The media type (MIME type).
+         * 
+         * @param {object} [options.certificatePolicy.x509CertificateProperties]
+         * Properties of the X509 component of a certificate.
+         * 
+         * @param {string}
+         * [options.certificatePolicy.x509CertificateProperties.subject] The subject
+         * name. Should be a valid X509 Distinguished Name.
+         * 
+         * @param {array} [options.certificatePolicy.x509CertificateProperties.ekus]
+         * The enhaunced key usage.
+         * 
+         * @param {object}
+         * [options.certificatePolicy.x509CertificateProperties.subjectAlternativeNames]
+         * The subject alternative names.
+         * 
+         * @param {array}
+         * [options.certificatePolicy.x509CertificateProperties.subjectAlternativeNames.emails]
+         * Email addresses.
+         * 
+         * @param {array}
+         * [options.certificatePolicy.x509CertificateProperties.subjectAlternativeNames.dnsNames]
+         * Domain names.
+         * 
+         * @param {array}
+         * [options.certificatePolicy.x509CertificateProperties.subjectAlternativeNames.upns]
+         * User principal names.
+         * 
+         * @param {array}
+         * [options.certificatePolicy.x509CertificateProperties.keyUsage] List of key
+         * usages.
+         * 
+         * @param {number}
+         * [options.certificatePolicy.x509CertificateProperties.validityInMonths] The
+         * subject alternate names.
+         * 
+         * @param {array} [options.certificatePolicy.lifetimeActions] Actions that
```

Comment:
Yes in the models: [https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/keyVault/lib/models/lifetimeAction.js](https://github.com/Azure/azure-sdk-for-node/blob/master/lib/services/keyVault/lib/models/lifetimeAction.js)


---
In reply to: [76497264](https://github.com/Azure/azure-sdk-for-node/pull/1858#discussion_r76497264) [](ancestors = 76497264)